x/executionlayer/types/errors.go

```diff
@@ -2,21 +2,23 @@ package types
 
 import (
 	"fmt"
+	"strings"
+
 	sdk "github.com/hdac-io/friday/types"
 )
 
 const (
 	DefaultCodespace sdk.CodespaceType = ModuleName
 
-	CodePublicKeyDecode      sdk.CodeType = 101
-	CodeProtocolVersionParse sdk.CodeType = 102
-	CodeTomlParse            sdk.CodeType = 103
-	CodeInvalidValidator     sdk.CodeType = 201
-	CodeInvalidDelegation    sdk.CodeType = 202
-	CodeInvalidInput         sdk.CodeType = 203
-	CodeInvalidAddress		 sdk.CodeType = sdk.CodeInvalidAddress
-	CodeGRpcExecuteMissingParent sdk.CodeType = 301
-	CodeGRpcExecuteDeployGasError sdk.CodeType = 302
+	CodePublicKeyDecode            sdk.CodeType = 101
+	CodeProtocolVersionParse       sdk.CodeType = 102
+	CodeTomlParse                  sdk.CodeType = 103
+	CodeInvalidValidator           sdk.CodeType = 201
+	CodeInvalidDelegation          sdk.CodeType = 202
+	CodeInvalidInput               sdk.CodeType = 203
+	CodeInvalidAddress             sdk.CodeType = sdk.CodeInvalidAddress
+	CodeGRpcExecuteMissingParent   sdk.CodeType = 301
```

Comment:
Is same error code okay here?

Reply:
https://github.com/hdac-io/friday/pull/147/commits/7c7de4a34d4c479b11be6756da3d89bbbb12551e